packages/webdriver/webdriver.d.ts

```diff
@@ -370,6 +370,8 @@ declare namespace WebDriver {
         headers?: {
             [name: string]: string;
         };
+        transformRequest?: (requestOptions: any) => any;
+        transformResponse?: (requestOptions: any, response: any) => any;
```

Comment:
`requestOptions` and `response` are pretty much well defined, especially for [response](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/got/index.d.ts#L313-L325). For `requestOptions` it would be great to have at least some decent type support.

Reply:
Thanks for looking into it! As described in the summary I've tried to add the typings from `@types/request` / `@types/got` but I can't figure out how I can reference those without an `import`. Is there a way to do it with `/// <reference types="requets" />` ? Or should I just copy over the typings into the definition files of webdriver?

Reply:
I think it should be pretty similar to how we do it e.g. in the Cucumber framework where we have the types as [a dependency](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cucumber-framework/package.json#L41) and import it in the [`*.d.ts` file](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cucumber-framework/cucumber-framework.d.ts#L1)